kitodo-mediaserver-fileserver/src/main/java/org/kitodo/mediaserver/fileserver/controller/FileController.java

```diff
@@ -92,11 +95,28 @@ public void getFile(
 
         } else {
             work = optionalWork.get();
+
             if (!work.isEnabled() && !StringUtils.endsWith(derivativePath, workId + ".xml")) {
                 String message = "Work with id " + workId + " is disabled";
                 LOGGER.info(message);
                 throw new HttpForbiddenException(message);
             }
```

Comment:
I always though of the network check as a replacement of enabled. Is there really a difference between enabled=true and allowedNetwork=global / enabled=false and allowedNetwork=none?
If we replace enabled, however, we must do it by the same rules as enabled, i.e. the mets/mods xml must always be delivered. And: the mets/mods xml should be reduces when allowed=none (as now by disablement).